statik/database.py

```diff
@@ -418,6 +418,14 @@ def __init__(self, model=None, session=None, **kwargs):
                     "Attempting to look up primary keys for ManyToMany " +
                     "field relationship: %s", self.field_values[field_name]
                 )
+
+                # check if field values contains duplicates
+                if len(self.field_values[field_name]) != len(set(self.field_values[field_name])):
+                    logger.warning("Duplicates found in field value array: %s",
+                                   self.field_values[field_name])
+                    # remove duplicate values by converting to set
+                    self.field_values[field_name] = set(self.field_values[field_name])
```

Comment:
Hey! Shouldn't this be converted back to a list?

Reply:
I meant the same thing, that the datatype be the same, but wouldn't using a set create conflicts with this https://github.com/thanethomson/statik/issues/76? Since, set doesn't preserve order.